feedback-module/src/translations/en/translation.js

```diff
@@ -65,15 +64,15 @@ export const TRANSLATIONS_EN = {
     title:
       "Thank you for your feedback! It has been submitted.<br /><br />Your feedback is anonymous and confidential, so you will not receive a reply.",
     plainText:
-      "The City of New York is always trying to improve its services. Are you interesed in being a user research participant? This is is entirely voluntary and you can opt out at any time. Signing up to be a user research participant will have no impact on your feedback today, or your eligibility to access or receive services in the future.",
+      "<p>The City of New York is always trying to improve its services. Are you interesed in being a user research participant? This is is entirely voluntary and you can opt out at any time. Signing up to be a user research participant will have no impact on your feedback today, or your eligibility to access or receive services in the future.</p>",
```

Comment:
Are we sure we want paragraph tags inside the translation files? I don't know if that would be considered best practice because it could interfere with the actual translation that goes on

Reply:
Yeah, I thought about this too, but I'm just not sure how else to customize the spacing between each paragraph of the body text (since design pointed out that there should be more spacing between each paragraph, but still keep it less than the spacing between the title and paragraph). I had considered just adding in another `<br />`, but we can't really customize how large that space is. I think another option would be to make the `plainText` an array, and then in `Module.js` map each piece of text into a `<p>` tag with our desired padding— not sure if this would be over-complicating things for our current purposes; it could become necessary if we ever end up having even more paragraphs per section in the future though. What do you think?

Reply:
I think we may want to find another solution. Additionally I don't know if we necessarily want <br /> within our translations files either. Maybe something we should ask Rapi about.

Reply:
Yeah, I definitely agree that we should ask Rapi about this. Which part are you referring to in your second sentence?

Reply:
Sorry I meant line breaks within the files either